endpoint/jobid.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/zrepl/zrepl/zfs"
 )
 
-// An instance of this type returned by MakeJobID guarantees
-// that that instance's JobID.String() can be used in a ZFS dataset name and hold tag.
+// JobID instances returned by MakeJobID() guarantee JobID.String() to
+// return a string that can be used in a ZFS dataset name and hold tag.
```

Comment:
`guarantee that JobID.String() will return a string` ...?

Reply:
Gave it another shot. If you dislike the revision I'll revert to the old "that that'.